docs/cep/query-guide/functions/unitconversion/MmTokm.md

```diff
@@ -6,16 +6,38 @@ This converts the input given in megameters into kilometers.
 
 ## Syntax
 
-    <DOUBLE> unitconversion:MmTokm(<INT|LONG|FLOAT|DOUBLE> p1)
+```sql
+<DOUBLE> unitconversion:MmTokm(<INT|LONG|FLOAT|DOUBLE> p1)
+```
 
 ## Query Parameters
 
-| Name | Description                                                           | Default Value | Possible Data Types   | Optional | Dynamic |
-|------|-----------------------------------------------------------------------|---------------|-----------------------|----------|---------|
+| Name | Description | Default Value | Possible Data Types   | Optional | Dynamic |
+|------|-------------|---------------|-----------------------|----------|---------|
 | p1   | The value that needs to be converted from megameters into kilometers. |               | INT LONG FLOAT DOUBLE | No       | Yes     |
 
 ## Example 1
 
-    unitconversion:MmTokm(1)
+```sql
+SELECT unitconversion:MmTokm(1) AS kilometers;
+```
 
-The megameter value `1` is converted into kilometers as `1000.0`.
+This query uses the `unitconversion:MmTokm()` function to convert the value of `1` megameter into its equivalent value in kilometers. The conversion result is `1000` kilometers.
+
+## Example 2
+
+```sql
+CREATE STREAM InputStream (distance_mm double);
+CREATE STREAM OutputStream (distance_mm double, distance_km double);
```

Comment:
@diana-macrometa We should make the Output Stream SINK STREAM so users can use the console to view the outputstream.. applicable everywhere we are using CREATE STREAM as the output stream

Reply:
In addition, `CREATE STREAM` is by default `SOURCE`. So, the SINK has to be explicitly defined.

Reply:
Well spotted, thank you!